tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+
+
+@pytest.fixture
+def mock_bot():
+    mock_bot = AsyncMock()
+    return mock_bot
+
+
+@pytest.fixture
+def mock_msg_fmt():
+    mock_msg_fmt = MagicMock()
+    mock_msg_fmt.bot_is_ready = MagicMock()
+    return mock_msg_fmt
+
+
+@pytest.fixture
+def mock_debug_fmt():
```

Comment:
Maybe this fixture needs to be on the unit test file. Do you think it will be used on another unit test? If not, let it on the unit test file, if yes, let it here.

Reply:
Not sure to be honest, let's leave it here until we finish the first version.